### ISSUE TITLE
Create a User : Global permissions -> Display of Access permissions order is changed

### ISSUE THREAD

**Issue by dasarinaidu** [rancher/dashboard#7153]
**Setup**
- Rancher version: v2.7-4092002-head
- Browser type & version: Chrome

**Describe the bug**
While creating a user -- User seeing the Global permissions are miss ordered / Not in the correct order

**To Reproduce**

1. Login as a admin user 
2. Create a user
3. On Create user screen Global Permissions
4. Check for the order of access permissions under 'Global Permissions'

**Actual:**
1. Display order is messed up, seeing a different order from v2.6.8
2. Screen shot attached

**Result**

**Expected Result**
1. Correct order should be displayed under Global Permissions

**Screenshots**
<img width="1487" alt="image" src="https://user-images.githubusercontent.com/22034884/195087470-11271f2f-8a7c-4a5b-90c8-0e0f98f74f9d.png">

**Additional context**

**Comment by richard-cox** [comment]
@dasarinaidu Can you confirm the order you expect to see? On my instance (same version) i see the correct order (below). Have either of your `ui-dashboard-index` or `ui-offline-preferred` settings been updated?

![image](https://user-images.githubusercontent.com/18697775/195099933-6a28f141-3675-4115-bad7-3b93c4fdf77b.png)

**Comment by dasarinaidu** [comment]
@richard-cox Thank you for looking into. The expected behavior I m looking is the same screen which you are seeing it on your end. 

I did not change any settings on my end (Screen as below) but I am seeing correct order with my new rancher setup (2.7-head). I will keep an eye on this ticket for some time and I will close it if I did not see issues with new rancher setup.

![image](https://user-images.githubusercontent.com/22034884/195154724-28d70eac-00f5-4ee2-aa44-8eece76c47f1.png)
